css_stylesheet.css

```diff
@@ -0,0 +1,70 @@
+body {
+	font-family: "Arial", sans-serif;
+	margin: 5px;
+	padding: 5px;
+	margin-left: auto;
+	margin-right: auto;
```

Comment:
Try not to use margin and margin-left/right. Combine into a single margin rule if you can